blueprints/resource/files/app/models/__name__.js

```diff
@@ -1,14 +1,15 @@
+import Ember from 'ember';
 import Resource from 'ember-jsonapi-resources/models/resource';
-import { attr, hasOne, hasMany, hasRelated } from 'ember-jsonapi-resources/models/resource';
+import { attr, hasOne, hasMany } from 'ember-jsonapi-resources/models/resource';
 
 export default Resource.extend({
-  type: '<%= entity %>'
+  type: '<%= entity %>',
+  service: Ember.inject.service('<%= resource %>'),
```

Comment:
This looks great.

I don't understand why the injection of the service from the initializer didn't work for the new instances. Do you?